src/createNumbersGenerator.js

```diff
@@ -25,8 +25,22 @@
  *
  * @return {number}
  */
+
 function createNumbersGenerator(numbers, min, max) {
-  // write code here
+  let rawNumbers = numbers;
+  if (rawNumbers === undefined) {
```

Comment:
add default values to the arguments

Reply:
changed this, as I told before, I thought we can't change parameters in braces.